### ISSUE TITLE
7.2.0 breaks Object.freeze

### ISSUE THREAD

**Issue by nihgwu** [yannickcr/eslint-plugin-react#1366]
the following code works on v7.1.0
``` js
  static propTypes = Object.freeze({
    value: PropTypes.any.isRequired,
  });
```

**Comment by ljharb** [comment]
Can you elaborate on "works"?

Do you have `Object.freeze` defined as a wrapper function in `settings`?

**Comment by nihgwu** [comment]
@ljharb "works" means I get no eslint errors
I've tried your suggestion, but still no fortune. still get the errors `error  'value' is missing in props validation          react/prop-types`
```json
  "settings": {
    "react": {
      "version": "15.6.1"
    },
    "propWrapperFunctions": [ "Object.freeze" ]
  },
```

**Comment by ljharb** [comment]
That seems like it's a bug in the `prop-types` rule then (without `propWrapperFunctions`, you should definitely get a warning there).

**Comment by yannickcr** [comment]
@nihgwu If I'm right this pattern was not detected in v7.1.0 and the propTypes validation was just ignored for the component. This is no longer the case since v7.2.0.

But https://github.com/yannickcr/eslint-plugin-react/issues/1366#issuecomment-322123945 should have worked, so there was definitively a bug here.

It should be fixed in next release.
